angularSTEP/src/app/view-profiles/view-profiles.component.ts

```diff
@@ -1,41 +1,24 @@
-import {
-  AngularFirestore,
-  AngularFirestoreCollection,
-} from '@angular/fire/firestore';
 import {Component, OnInit, NgZone} from '@angular/core';
+import * as algoliasearch from 'algoliasearch/lite';
+import {environment} from '../../environments/environment'
 
-import {AngularFireAuth} from '@angular/fire/auth';
-import {Observable} from 'rxjs';
 import {Router} from '@angular/router';
-import {User} from '../user';
+
+const searchClient = algoliasearch(environment.algolia.appId, environment.algolia.apiKey);
```

Comment:
Here is where you will want to put the appId and apiKey into the environment folder

After the firebase configurations put
`algolia: { appId: '{app id here}', apiKey: '{app key here}' }`